index.html

```diff
@@ -207,36 +208,47 @@ <h2>Legend</h2>
        return p;
      }
 
-     function sanitizeHTML(value) {
-       // Only support links and code.
-       const template = document.createElement('template');
-       const div = template.content.ownerDocument.createElement('div');
-       template.content.append(div); // force parser context to "in body"
-       div.innerHTML = value;
-       const iterator = document.createNodeIterator(div);
-       let node = iterator.nextNode(); // This is the div
-       while (node = iterator.nextNode()) {
-         if (node.nodeType === Node.TEXT_NODE) {
-           continue;
+     function parseMarkdown(markdown, enableLinks = true) {
```

Comment:
I'm not a fan of defaults being true.  Though see below for an alternative structure.